jest/private/jest_config_template.mjs

```diff
@@ -113,11 +119,19 @@ if (isTest && autoConfTestSequencer) {
 // If this is an update snapshot target the configure the Bazel snapshot resolver
 if (updateSnapshots) {
   if (config.snapshotResolver) {
-    process.env.JEST_TEST__USER_SNAPSHOT_RESOLVER = path.isAbsolute(
-      config.snapshotResolver
-    )
+    const snapshotResolverPath = path.isAbsolute(config.snapshotResolver)
       ? config.snapshotResolver
-      : path.resolve(__dirname, config.snapshotResolver);
```

Comment:
What required changing this?

Reply:
It is because its a `.mjs` file and `__dirname` isn't valid in that context. I think it just landed broken tho I'm not sure how because there is an e2e test that runs this update code

Reply:
Ohhhh. this code path isn't covered by the e2e because it is the custom resolver path. I'll add an e2e test.